api/services/User/index.js

```diff
@@ -39,33 +39,36 @@ export default class User extends BaseModelService {
   }
 
   static async findAllAvailabilityDuring(keys) {
-    return Promise.all(
-      keys.map(
-        async (key) =>
-          !(await Models.CalendarEventInvite.scope('visible').findOne({
+    return Models.CalendarEventInvite.scope('visible')
+      .findAll({
+        where: {
```

Comment:
This will possibly get the pairs where user_id matches with the one key and the date time range with another. Move this to inside the include clause and use sequelize column reference func to refer to this very column.

In the include where clause, use the Or operator and provide options for each key. Each option item should include 3 checks. user_id by referring to the column here, overlap date time range, and exclude event ids.

This same pattern should be used for the other PR too. Lets solve these 2 issues one by one. 

Lets resolve this one first and then move to the next.